extensions/WSLExtension/Contracts/IWslManager.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using WSLExtension.DistributionDefinitions;
+using WSLExtension.Models;
+
+namespace WSLExtension.Contracts;
+
+/// <summary>
+/// Used to interact between the WSL dev environment compute systems and the WSL mediator.
+/// </summary>
+public interface IWslManager

```

Comment:
Looks to me that all methods act on a Distribution.  Maybe this can be renamed to `IDistributionManager`?  Then all methods can drop `Distribution` from their name.

Reply:
In the future this will/can have more methods so will leave as is for now.